client/src/game/rotatePlayer.js

```diff
@@ -0,0 +1,9 @@
+export default function rotatePlayer(keyCode, player) {
+  if (keyCode === "ArrowRight") {
+    player.direction = "RIGHT";
+  } else if (keyCode === "ArrowLeft") {
+    player.direction = "LEFT";
+  } else if (keyCode === "Space") {
+    player.direction = "JUMPING";
+  }
```

Comment:
If you like the idea of https://github.com/JoscaWij/Spacey/pull/33#discussion_r489473495, this would be another case for an object map.

Reply:
Hopefully I implemented it the right way now 😄